test/e2e/multik8s/suite_test.go

```diff
@@ -49,47 +56,164 @@ func waitControllerToBeReady() {
 	})
 }
 
+func setupEnvironment(namespace, pvcName string) {
+	GinkgoHelper()
+	By("setting up the environment")
+	Eventually(func() error {
+		return createNamespace(primaryK8sCluster, namespace)
+	}).Should(Succeed())
+	Eventually(func() error {
+		return createNamespace(secondaryK8sCluster, namespace)
+	}).Should(Succeed())
+	Eventually(func() error {
+		return applyRBDPoolAndSCTemplate(primaryK8sCluster, cephClusterNamespace)
+	}).Should(Succeed())
+	Eventually(func() error {
+		return applyRBDPoolAndSCTemplate(secondaryK8sCluster, cephClusterNamespace)
+	}).Should(Succeed())
+	Eventually(func() error {
+		return applyPVCTemplate(primaryK8sCluster, namespace, pvcName)
+	}).Should(Succeed())
+}
+
+func writeRandomDataToPV(ctx context.Context, namespace, pvcName string) string {
+	GinkgoHelper()
+	By("writing some random data to PV(C)")
+	writeJobName := util.GetUniqueName("job-")
+	Eventually(ctx, func() error {
+		return applyWriteJobTemplate(primaryK8sCluster, namespace, writeJobName, pvcName)
+	}).Should(Succeed())
+	Eventually(ctx, func(g Gomega) {
+		job, err := getJob(primaryK8sCluster, namespace, writeJobName)
+		g.Expect(err).NotTo(HaveOccurred())
+		g.Expect(IsJobConditionTrue(job.Status.Conditions, batchv1.JobComplete)).To(BeTrue())
+	}).Should(Succeed())
+	stdout, _, err := kubectl(primaryK8sCluster, nil, "logs", "-n", namespace, "job/"+writeJobName)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(len(stdout)).NotTo(Equal(0))
+	return string(stdout)
+}
+
+func createMantleBackup(namespace, pvcName, backupName string) {
+	GinkgoHelper()
+	By("creating a MantleBackup object")
+	Eventually(func() error {
+		return applyMantleBackupTemplate(primaryK8sCluster, namespace, pvcName, backupName)
+	}).Should(Succeed())
+}
+
+func waitMantleBackupSynced(namespace, backupName string) {
+	GinkgoHelper()
+	By("checking MantleBackup's SyncedToRemote status")
+	Eventually(func() error {
+		mb, err := getMB(primaryK8sCluster, namespace, backupName)
+		if err != nil {
+			return err
+		}
+		if !meta.IsStatusConditionTrue(mb.Status.Conditions, mantlev1.BackupConditionSyncedToRemote) {
+			return errors.New("status of SyncedToRemote condition is not True")
+		}
+		return nil
+	}, "10m", "1s").Should(Succeed())
+}
+
+func ensureTemporaryResourcesRemoved(ctx context.Context) {
```

Comment:
```suggestion
func ensureTemporaryResourcesDeleted(ctx context.Context) {
```

In k8s context, `delete` is more appropriate than `remove`.

Reply:
Fixed in https://github.com/cybozu-go/mantle/pull/81/commits/88a6ddb64e123ada38b03c8e665f8c470a82df3b.